au4/src/project/internal/projectactionscontroller.cpp

```diff
@@ -149,9 +151,55 @@ bool ProjectActionsController::isFileSupported(const muse::io::path_t& path) con
 
 bool ProjectActionsController::closeOpenedProject(bool quitApp)
 {
-    //! TODO AU4
-    NOT_IMPLEMENTED;
-    return false;
+    if (m_isProjectClosing) {
+        return false;
+    }
+
+    m_isProjectClosing = true;
+    DEFER {
+        m_isProjectClosing = false;
+    };
+
+    IAudacityProjectPtr project = globalContext()->currentProject();
+    if (!project) {
+        return true;
+    }
+
+    bool result = true;
+
+    // if (project->needSave().val) {
+    //     IInteractive::Button btn = askAboutSavingScore(project);
+
+    //     if (btn == IInteractive::Button::Cancel) {
+    //         result = false;
+    //     } else if (btn == IInteractive::Button::Save) {
+    //         result = saveProject();
+    //     } else if (btn == IInteractive::Button::DontSave) {
+    //         result = true;
+    //     }
+    // }
```

Comment:
need add issue

Reply:
You already did it :)
https://github.com/audacity/audacity/issues/6664